Dockerfile.tests

```diff
@@ -0,0 +1,27 @@
+FROM python:3.6
+
+COPY requirements.txt /requirements.txt
+
+RUN pip download $(grep ^ansible /requirements.txt) --no-cache-dir --no-deps -d /tmp/pip && \
+    mkdir /tmp/ansible && \
+    tar -C /tmp/ansible -xf /tmp/pip/ansible* \
+    && mv /tmp/ansible/ansible* /ansible \
```

Comment:
Minor note: let's put all `&&` at the end on all lines for consistency.

Reply:
Done